test/integration/backend-basic/fixture/toolpad/pages/basic/page.yml

```diff
@@ -151,6 +151,10 @@ spec:
           $$jsExpressionAction: |-
             await setCustomCookie.call()
             context.refetch()
+    - component: Checkbox
```

Comment:
This is not showing up in the screenshot. I don't believe the visual regression tests are using this fixture

Reply:
I follow this [link](https://github.com/mui/mui-toolpad/pull/2494#issuecomment-1698071572) 
and I drag a componen into the editor

Reply:
Are you sure you used that exact command? The change in this PR is under test/integration/backend-basic, I'd expect it to be under test/visual/components.